spec/events/inventory_aggregate_spec.rb

```diff
@@ -462,38 +462,6 @@
         }
       ))
     end
-
-    it "should process a snapshot event" do
```

Comment:
Should this be kept, or replaces with one that is an event-driven snapshot create?

Reply:
Eh, I guess the one below covers us